.github/workflows/build-and-publish.yaml

```diff
@@ -0,0 +1,136 @@
+      - name: Set Output
+        id: set-output
+        run: echo "::set-output name=digest::$(docker inspect --format='{{index .RepoDigests 0}}' ${IMAGE_REGISTRY}/quay-builder-qemu:latest)"
+
+  operator-image:
+    name: Publish 3.5 Operator Image
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          ref: 'redhat-3.5'
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: projectquay+quay_github
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: quay.io/projectquay/quay-operator:3.5
+
+  operator-index-images:
+    name: Publish 3.5 Catalog Index Image
+    runs-on: 'ubuntu-latest'
+    needs: [quay-image, clair-image, builder-image, qemu-builder-image]
+    env:
+      INDEX_TAG: quay.io/projectquay/quay-operator-index:3.5
+      TAG: 3.5
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          ref: 'redhat-3.5'
+
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: projectquay+quay_github
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Update CSV Image Versions
+        uses: mikefarah/yq@master
+        with:
+          cmd: |
+            yq eval -i '
+              .metadata.annotations.quay-version = strenv(TAG) |
+              .metadata.annotations.containerImage = "quay.io/projectquay/quay-operator:"+env(TAG) |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].image = "quay.io/projectquay/quay-operator:"+env(TAG) |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[4].value = "${{needs.quay-image.outputs.digest}}" |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[5].value = "${{needs.clair-image.outputs.digest}}" |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[6].value = "${{needs.builder-image.outputs.digest}}" |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[7].value = "${{needs.qemu-builder-image.outputs.digest}}" |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[8].value = "$centos/postgresql-10-centos7@sha256:de1560cb35e5ec643e7b3a772ebaac8e3a7a2a8e8271d9e91ff023539b4dfb33" |
+              .spec.install.spec.deployments[0].spec.template.spec.containers[0].env[9].value = "centos/redis-32-centos7@sha256:06dbb609484330ec6be6090109f1fa16e936afcf975d1cbc5fff3e6c7cae7542"
+              ' deploy/manifests/quay-operator/"${TAG}"/quay-operator.clusterserviceversion.yaml
+
+      - name: Build Catalog Index
```

Comment:
The image built in this step is the `Catalog Bundle Image` or `bundle image`. In order to create the `index image`, we would have to use the `opm index add --bundles` command. The [readme doc](https://github.com/quay/quay-operator) (from step 4 - till end) has the steps.

Reply:
As far as I understand that's only the case for the new bundle model we're using from 3.6. I followed the [README for 3.5](https://github.com/quay/quay-operator/tree/redhat-3.5#local-development) (see `Building custom CatalogSource` section) to build the package manifest under `deploy`, which seems to entirely skip using `opm`. Did I miss something?

Reply:
had to change this, turns out the CSV on `redhat-3.5` isn't working... I suppose we don't we it for anything? 
marking this as resolved as I'm doing as suggested and using opm index.

Reply:
I am not aware how the operator was deployed prior to `3.6`. I was able to use `opm` cli to create the index image for a 3.5.8 operator(using the CSV for redhat-3.5). Let's follow up the conversation on [slack](https://coreos.slack.com/archives/CGJC7J7KR/p1632411594250300)

Reply:
I just figured this out with Ricardo's help @harishsurf. It works for you because we're using the bundle format for downstream. https://github.com/quay/quay-operator/pull/538 makes the same (or similar enough anyway) changes to `redhat-3.5` so that we can use the `opm` to build upstream too :)